src/components/Form.jsx

```diff
@@ -0,0 +1,53 @@
+import React from "react";
+import Button from "./Button.jsx";
+import Message from "./Messege.jsx";
+import "./styles.css";
+export default class Form extends React.Component {
+  state = { name: "", birthday: null, buttClick: false };
+
+  handleChange = event => {
+    this.setState({ [event.target.name]: event.target.value });
+  };
+
+  change = () => {
```

Comment:
name "change" to this function doesn't represent what exactly is it doing, rename